osu.Game/Overlays/Profile/Sections/Recent/DrawableRecentActivity.cs

```diff
@@ -216,15 +215,15 @@ private void createMessage()
             rulesets.AvailableRulesets.FirstOrDefault(r => r.ShortName == activity.Mode)?.Name ?? activity.Mode;
 
         private void addUserLink()
-            => content.AddLink(activity.User?.Username, LinkAction.OpenUserProfile, getLinkArgument(activity.User?.Url), creationParameters: t => t.Font = getLinkFont(FontWeight.Bold));
+            => content.AddLink(activity.User.AsNonNull().Username, LinkAction.OpenUserProfile, getLinkArgument(activity.User.AsNonNull().Url), creationParameters: t => t.Font = getLinkFont(FontWeight.Bold));
```

Comment:
Similarly here, none of this could *actually* ever be null, or the activity would look broken.